### PR TITLE
More doc comment additions

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Summary of the changes (Less than 80 chars)
 - Detail 1
 - Detail 2

Addresses #bugnumber (in this specific format)

### CODE REVIEW

File: src/Hosting/Abstractions/src/IWebHost.cs [hunk 1]
```diff
@@ -31,13 +31,15 @@ public interface IWebHost : IDisposable
         /// <summary>
         /// Starts listening on the configured addresses.
         /// </summary>
+        /// <param name="cancellationToken">used to abort program start.</param>
```
Comment (Eilon) [inline src/Hosting/Abstractions/src/IWebHost.cs]: Uppercase the first word?

File: src/Http/Http.Extensions/src/UriHelper.cs [hunk 2]
```diff
@@ -171,16 +171,16 @@ public static string Encode(Uri uri)
...
         /// <param name="request">The request to assemble the uri pieces from.</param>
-        /// <returns></returns>
+        /// <returns>The encoded string version of the url from <paramref name="request"/>.</returns>
         public static string GetEncodedUrl(this HttpRequest request)
         {
             return BuildAbsolute(request.Scheme, request.Host, request.PathBase, request.Path, request.QueryString);
         }
         /// <summary>
-        /// Returns the relative url 
+        /// Returns the relative url
```
Comment (Eilon) [inline src/Http/Http.Extensions/src/UriHelper.cs]: Might as well fix this: `url` --> `URL.` (All uppercase, add a period.)

Comment (Eilon) [reply src/Http/Http.Extensions/src/UriHelper.cs]: Or make it `URI`?

File: src/Http/Http.Extensions/src/UriHelper.cs [hunk 3]
```diff
@@ -171,16 +171,16 @@ public static string Encode(Uri uri)
...
         /// </summary>
         /// <param name="request">The request to assemble the uri pieces from.</param>
```
Comment (Eilon) [inline src/Http/Http.Extensions/src/UriHelper.cs]: Everywhere that `uri`, `Uri`, `url`, `Url` appear in "prose", they must be all lowercase. Only when it's referencing an identifier (property, type name, etc.), it should be PascalCase.

Comment (ryanbrandenburg) [reply src/Http/Http.Extensions/src/UriHelper.cs]: I assume you mean uppercase, not lowercase?

**Comment by ryanbrandenburg** [comment]
:up::date:

File: src/Http/Http.Extensions/src/UriHelper.cs (hunk 3 above)
Comment (Eilon) [reply src/Http/Http.Extensions/src/UriHelper.cs]: Sorry yes!

**Review by Eilon — APPROVED** [review]
Looks good to me.

**Comment by ryanbrandenburg** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=246339

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2577

**Pull request merged**